conf/messages.en

```diff
@@ -1016,15 +1017,9 @@ authorisedOfficialPreviousAddress.confirmPage.changeLinkText = Change authorised
 
 authorisedOfficialPreviousAddress.checkYourAnswersLabel = Previous home address
 
-# You have added one Authorised Official Page Messages
+# You have added Authorised Official Page Messages
 # ----------------------------------------------------------
-addedOneAuthorisedOfficial.checkYourAnswers.title = You have added one authorised official
-addedOneAuthorisedOfficial.checkYourAnswers.heading = Check the authorised official’s details
-
-# You have added second Authorised Official Page Messages
-# ----------------------------------------------------------
-addedSecondAuthorisedOfficial.checkYourAnswers.heading = Check the second authorised official’s details
-addedSecondAuthorisedOfficial.checkYourAnswers.title = Check the second authorised official’s details
+addedOneAuthorisedOfficial.checkYourAnswers.title = Check the authorised official’s details
```

Comment:
for absolute clarity, would you be able to rename this key to 'addedAuthorisedOfficial'? not a priority - if it's gonna take longer than 15 minutes, keep it as is